rbd/rbd.go

```diff
@@ -813,6 +818,7 @@ func (image *Image) Discard(ofs uint64, length uint64) (int, error) {
 	return int(ret), nil
 }
 
+// ReadAt copies data from the image into the supplied buffer.
```

Comment:
Missing "implements rbd_read".

Reply:
I'm not forcing "Implements" where it was already missing, since my goal right now is to make revive happier. I'd like to continue skipping it for now.

PS.  One of my (barely started) side projects is to make source code checker specific for this project that can understand our comment style and produce  reports, etc. I'll be more motivated to go back and add "Implements" where it is missing when I resume work on that.

Reply:
Ok!